File: src/main/java/com/my/stock/stockmanager/dto/stock/DashboardStock.java
package com.my.stock.stockmanager.dto.stock;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DashboardStock {
	private String symbol;
	private String code;
	private String national;
	private String name;
	private Double avgPrice;
	private Double quantity;
	private Double priceImportance;
	// 주식 현재가
	private Double nowPrice;

	private String rateOfReturnPer;
}

File: domain/src/main/kotlin/kr/pe/hws/stock/constants/stock/DashboardStock.kt
package kr.pe.hws.stock.constants.stock

import java.math.BigDecimal

data class DashboardStock(
    val symbol: String,
    val code: String,
    val national: String,
    val name: String,
    val avgPrice: BigDecimal,
    val quantity: BigDecimal,
    val totalDividend: BigDecimal,
    val priceImportance: BigDecimal,
    val nowPrice: BigDecimal,
    val compareToYesterdaySign: String,
    val compareToYesterday: BigDecimal,
    val rateOfReturnPer: String,
)
